packages/react-lowcode/src/codegen/generation/generators/list/mui-datatable-generator.ts

```diff
@@ -98,6 +99,80 @@ export default class MuiDataTableGenerator implements TableGenerator
       return alteredSource
     }
 
+    async getColumnSourcePosition(position: SourceLineCol,
```

Comment:
please rename first argument to be more specific: `tablePosition` (it will be explicit - otherwise I could think of what is the position...)

please check also other coccurences of this type of parameter

Reply:
fixed